src/mozillavpn.cpp

```diff
@@ -896,10 +909,20 @@ void MozillaVPN::mainWindowLoaded() {
 #endif
 }
 
-void MozillaVPN::telemetryPolicyCompleted() {
-  logger.debug() << "telemetry policy completed";
-
+void MozillaVPN::onboardingCompleted() {
   SettingsHolder* settingsHolder = SettingsHolder::instance();
+
+  if (Feature::get(Feature::Feature_newOnboarding)->isSupported()) {
+#if !defined(MZ_ANDROID) && !defined(MZ_IOS)
+    logger.debug() << "onboarding completed";
+    // If new onboarding is turned off in the future, don't make them go through
+    // old onboarding (post auth + telemetry)
```

Comment:
This comment is a bit confusing to me. If the new onboarding is turned off, won't the feature check on line 915 prevent this block from being executed?

Reply:
We're making sure that when the new onboarding is completed, we also set the old onboarding (aka `postAuthenticationShown`) to completed, that way if the feature is turned off at some point, the user doesn't see the old onboarding.

Does that make sense?

Reply:
Thanks for the explanation. Rephrasing the comment to something like the following may help the reader:
"Mark the old onboarding expereince as completed as well, ensuring that users do not have to go through it if the new onboaring feature is turned off"